functions.js

```diff
@@ -0,0 +1,98 @@
+////////////////////////////////////////
+//         Helper Function            //
+////////////////////////////////////////
+
+
+export const hexToRgb = (hex) => {
+  var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
+  result = `rgb(${parseInt(result[1],16)},${parseInt(result[2],16)},${parseInt(result[1],16)})`
+  return result;
+}
+
+export const getRgbFromInput = (rgbString) => {
+
+    let red;    
+    let green;
+    let blue;
+
+    // Removing "rgb(" and ")" from the string "rgb(v1, v2, v3)"
+    let tempValue = rgbString.replace('rgb(', '');
+    tempValue = tempValue.replace(')','');
+    // console.log(tempValue);
+    const rgbVal = tempValue.split(',');
+    red = rgbVal[0];
+    green = rgbVal[1];
+    blue = rgbVal[2];
+    return({r:rgbVal[0], g:rgbVal[1], b:rgbVal[2]});
```

Comment:
consider using the variables like this

```suggestion
    // Removing "rgb(" and ")" from the string "rgb(v1, v2, v3)"
    let tempValue = rgbString.replace('rgb(', '');
    tempValue = tempValue.replace(')','');
    // console.log(tempValue);
    const rgbVal = tempValue.split(',');
    const red = rgbVal[0];
    const green = rgbVal[1];
    const blue = rgbVal[2];
    return({r: red, g: green, b: blue});
```